app/src/renderer/connectors/node.js

```diff
@@ -2,17 +2,17 @@
 const RestClient = require('./lcdClient.js')
 const mockedRestClient = require('./lcdClientMock.js')
 const RpcWrapper = require('./rpcWrapper.js')
-const mockedRpcWrapper = require('./rpcWrapperMock.js')
+const MockedRpcWrapper = require('./rpcWrapperMock.js')
```

Comment:
what is our standard for capital vs camel casing?

Reply:
I used capital here, as it is a constructor-like function.

Reply:
In general only classes and constants should have capital letters